src/test/scala/org/broadinstitute/dsde/firecloud/dataaccess/MockRawlsDAO.scala

```diff
@@ -200,7 +200,7 @@ class MockRawlsDAO extends RawlsDAO {
   private val rawlsWorkspaceWithAttributes = WorkspaceDetails(
     "attributes",
     "att",
-    "id",
+    "00000000-0000-0000-0000-000000000000",
```

Comment:
I needed to update this example test data; `UUID.fromString("id")` would cause problems. This is more realistic test data anyway.